homeassistant/components/zeversolar_local/translations/en.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Run the following script to generate this file:
```
python3 -m script.translations develop
```

Reply:
https://github.com/home-assistant/core/pull/55440/commits/1059f6abd39e241fa448b0336ecb0a02fe43dd1e